test/functionality/typing/constant-type-inference.test.ts

```diff
@@ -0,0 +1,33 @@
+import { beforeAll, describe, expect, test } from 'vitest';
+import { retrieveNormalizedAst, withShell } from '../_helper/shell';
+import type { NormalizedAst } from '../../../src/r-bridge/lang-4.x/ast/model/processing/decorate';
+import { RDataType, TypeInferencer } from '../../../src/typing/type-inferencer';
+
+describe.sequential('Infer types for constant expressions', withShell(shell => {
+	let logicalAst: NormalizedAst | undefined;
+	let numberAst: NormalizedAst | undefined;
+	let stringAst: NormalizedAst | undefined;
+	const inferencer = new TypeInferencer();
+
+	beforeAll(async() => {
+		logicalAst = await retrieveNormalizedAst(shell, 'TRUE');
```

Comment:
Doing it this way for a quick test is perfectly fine! 
For the future we may want to separate these parse calls and potentially just move them to the respective test which then becomes a wrapper function similar to `assertDataflow` etc.